ddpui/core/ai/smart_chat_processor.py

```
@@ -0,0 +1,746 @@
+            # Add helpful suggestions for data queries if appropriate
+            suggestions = self._generate_query_suggestions(analysis, dashboard_context)
+
+            content = response.content
+            if suggestions:
+                content += "\n\n" + "\n".join(suggestions)
+
+            return EnhancedChatResponse(
+                content=content, intent_detected=analysis.intent, recommendations=suggestions
+            )
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)